ui/site/src/main.js

```diff
@@ -858,13 +858,19 @@ lichess.topMenuIntent = function() {
       },
       _renderUser: function(user) {
         var icon = '<i class="is-green line' + (user.patron ? ' patron' : '') + '"></i>';
-        var name = lichess.fp.contains(user.name, ' ') ? user.name.split(' ')[1] : user.name;
+        var name = user.name;
+        var titleTag = ''
+        if (lichess.fp.contains(user.name, ' ')) { 
+            var split = user.name.split(' ');
+            titleTag = '<span class = "title" data-title=' + split[0] + '>' + split[0] + '</span>&nbsp;';
```

Comment:
No spaces for HTML attributes, please (`class_=`). And some quotes (`"`) around data-title, just to be consistent.

Reply:
data-title is no longer a thing, only data-bot

Reply:
done 👍 
also now uses data-bot ![image](https://user-images.githubusercontent.com/45638668/49822971-6a194300-fd33-11e8-9c50-cd6bf7bc25a6.png)